ambry-api/src/main/java/com.github.ambry/config/StatsManagerConfig.java

```diff
@@ -34,15 +34,15 @@
   public final long publishPeriodInSecs;
 
   /**
-   * The option to enable or disable publishing stats.
+   * The time period in seconds to delay before the first stats collection is triggered
```

Comment:
Might be good to mention it in the comment or rename to initial.delay.upper.bound.in.secs to reflect the fact that the actual delay is randomly chosen between 0 and this value (exclusive).